index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>

```

Comment:
You should move this `index.html` to the folder `Exercise-HTML-CSS-1/`. Just drag and drop it!
Also for the `styles.css` folder, you should create a folder named `css` inside the `Exercise-HTML-CSS-1/` folder, and then move that file there. The resulting route will be `Exercise-HTML-CSS-1/css/styles.css`. You will later need to update your links in the `index.html` where you reference the styles file